src/vaadin-combo-box-mixin.html

```diff
@@ -696,6 +699,8 @@
             this._selectItemForValue(customValue);
             this.value = customValue;
           }
+        } else if (!this.allowCustomValue && !this.opened && itemsMatchedByLabel.length == 1) {
```

Comment:
We probably should test how it behaves with `allowCustomValue`. 🤔

Reply:
If I remove the `!allowCustomValue`, this test fails:
https://github.com/vaadin/vaadin-combo-box/blob/5007fcc182ed59fb1813e79e24349612561a781b/test/vaadin-combo-box.html#L204-L220

![combobox_custom_value_test_error](https://user-images.githubusercontent.com/28783969/96978796-6d80d680-1527-11eb-8916-01e2e80b3363.png)